subprojects/maven/src/main/java/org/gradle/api/publish/maven/internal/publication/DefaultMavenPom.java

```diff
@@ -65,6 +66,7 @@
     private MavenPomCiManagement ciManagement;
     private MavenPomDistributionManagementInternal distributionManagement;
     private final List<MavenPomMailingList> mailingLists = new ArrayList<MavenPomMailingList>();
+    private MapProperty<String, String> properties;
```

Comment:
:o: All the `Property` fields in this class should be `final`.

Reply:
Good catch, I forgot this when I removed the setter!